### ISSUE TITLE
simlayer forcing modifier key up

### ISSUE THREAD

**Issue by johnlindquist** [yqrashawn/GokuRakuJoudo#91]
With the following config, press and hold the keys in this order:

**a, d, f, j**

- Bug: When using a `simlayer`, `shift(d)` is forced `up` when `option(f)` is pressed:
- Expected: `shift(d)` should stay down when `option(f)` is pressed.

The behavior of `layers` below is what I expect the behavior of `simlayers` to be. Please note in the screenshots the different behavior of `shift`.

> Note, the configs are identical other than `a-mode` being swapped between `simlayers` and `layers`:

![simlayer behavior](https://user-images.githubusercontent.com/36073/97922690-17c3df80-1d1a-11eb-8fbf-7b7ee0c9d0a2.png)
```edn
{:simlayers {;
             :a-mode {:key :a}
             ;
             }
 :main [{:des "movement"
         :rules [:a-mode

                 [:##d :left_shift]
                 [:##f :left_option]


                 [:##h :left_arrow]
                 [:##j :down_arrow]
                 [:##k :up_arrow]
                 [:##l :right_arrow]
                 ;
                 ]}
        ;
        ]}
```

![layer behavior](https://user-images.githubusercontent.com/36073/97922600-f9f67a80-1d19-11eb-968f-e00abadf2bdd.png)
```edn
{:layers {;
             :a-mode {:key :a}
             ;
             }
 :main [{:des "movement"
         :rules [:a-mode

                 [:##d :left_shift]
                 [:##f :left_option]


                 [:##h :left_arrow]
                 [:##j :down_arrow]
                 [:##k :up_arrow]
                 [:##l :right_arrow]
                 ;
                 ]}
        ;
        ]}
```

**Comment by yqrashawn** [comment]
This is solved in version 0.3.3 which just released in homebrew.

**Comment by johnlindquist** [comment]
@yqrashawn confirmed fixed. THANK YOU SO MUCH!

![](https://media4.giphy.com/media/13qctMBrrgbwJi/giphy.gif)
